{{ cookiecutter.project_slug }}/scripts/deploy.sh

```diff
@@ -6,6 +6,14 @@ SERVICE_NAME=""
 AWS_CLUSTER_NAME=""
 AWS_RESOURCE_NAME=""
 TAG=""
+
+# validate if $1 variable exists if not, it means it is a tagged build, without circle branch variable
```

Comment:
I'm not a big fan of this approach, take a look [at the discussion here.](https://github.com/CheesecakeLabs/django-drf-boilerplate/pull/175#discussion_r510430535)

Reply:
@rodolfolottin @rkpinheiro this is probably related to your last changes, but not sure which.